app.py

```diff
@@ -3,12 +3,16 @@
 from chalicelib.s3 import S3Client
 from chalicelib.utils import get_file_extension_from_base64
 
+# Blueprints
+from chalicelib.blueprints.announcements import announcements_routes
+
 import uuid
 
 from chalicelib.api.listings import listings_api
 from chalicelib.api.applicants import applicants_api
 
 app = Chalice(app_name="zap")
+app.register_blueprint(announcements_routes)
```

Comment:
@cwgough Could you try look at the `chalicelib/api` directory and move the blueprint to there?